.github/workflows/new_theme_bundle.yml

```diff
@@ -0,0 +1,39 @@
+name: New theme bundle
```

Comment:
⚠️ for consistency with other workflow file, I suggest that, in the file name, we use dash (`-`) instead of underscore (`_`).
About the name of the file and the name of the workflow I suggest `Update Theme Bundle` which is clearer IMHO

Reply:
ok